javaslang-benchmark/src/test/java/javaslang/BenchmarkPerformanceReporter.java

```diff
@@ -223,9 +223,9 @@ private String calculatePerformanceStr(TestExecution result, Array<String> alter
             final Array<TestExecution> alternativeResults = resultsByKey.get(aggregateKey).getOrElse(Array::empty);
 
             return alternativeImplementations.map(altImpl -> Tuple.of(altImpl, alternativeResults.find(r -> altImpl.equals(r.getImplementation()))))
-                    .map(alt -> Tuple.of(alt._1, calculateRatioStr(result, alt._2)))
-                    .map(alt -> padRight(alt._2, altImplColSize(alt._1)))
-                    .mkString("  ");
+                                             .map(alt -> Tuple.of(alt._1, calculateRatioStr(result, alt._2)))
+                                             .map(alt -> padRight(alt._2, altImplColSize(alt._1)))
+                                             .mkString("  ");
```

Comment:
No, sorry. this does not look right. If you want column-alignment spring for a new-line (like everyone else does):

```java
return alternativeImplementations
        .map(altImpl -> Tuple.of(altImpl, alternativeResults.find(r -> altImpl.equals(r.getImplementation()))))
        .map(alt -> Tuple.of(alt._1, calculateRatioStr(result, alt._2)))
        .map(alt -> padRight(alt._2, altImplColSize(alt._1)))
        .mkString("  ");
```